cdk/lib/__snapshots__/prism-access.test.ts.snap

```diff
@@ -90,7 +90,7 @@ Object {
           },
           Object {
             "Key": "gu:cdk:version",
-            "Value": "0.37.0",
+            "Value": "0.38.0",
```

Comment:
N.B. since this is the only change to this template, I don't intend to update this stack set. Let me know if you think this is necessary.